.github/workflows/check_code_format.yml

```diff
@@ -19,6 +19,10 @@ jobs:
           fetch-depth: 0
 
       - uses: jiro4989/setup-nim-action@v2
+        with:
+          nim-version: stable
```

Comment:
I am not sure if this is needed.
```suggestion
```

Reply:
Yes it isn't. I just prefer writing it down for future changes.